platform/msm8974/platform.c

```diff
@@ -129,8 +129,8 @@ int platform_is_8974Pro()
 	return ret;
 }
 
-/* Check for 8974PRO AC chip */
-int platform_is_8974ac()
+/* Check for 8974 PRO chip */
+int platform_is_8974Pro()
```

Comment:
I'm not sure if this change is unsafe as-is or if the original impl was bugged, but this would change how usb_ss phy is inited/reset on pro-not-ac I think..

https://github.com/msm8916-mainline/lk2nd/blob/master/platform/msm_shared/usb30_udc.c#L116-L155

But I'm somewhat inclined to think it was their mistake so this is a reasonable change